### PR TITLE
Added 2 extractors and reworked vidstream extractor

### PR DESCRIPTION
**Pull request by Blatzar**
Was unsure where to put the extractor config. Maybe adding "extractorconfig" instead?

### CODE REVIEW

File: anime_downloader/config.py [hunk 1]
```diff
@@ -82,6 +82,9 @@
         },
         'animedaisuki': {
             "servers": ["official"]
+        },
+        'vidstream': {
+            "servers": ["vidstream","gcloud","mp4upload","cloud9","hydrax"]
```
Comment (vn-ki) [inline anime_downloader/config.py:87]: I don't like. Why are these needed?

File: anime_downloader/extractors/vidstream.py [hunk 2]
```diff
@@ -1,42 +1,70 @@
...
 
 class VidStream(BaseExtractor):
     def _get_data(self):
-        QUALITIES = {
-        "360":[],
-        "480":[],
-        "720":[],
-        "1080":[],
+
+        links = {
```
Comment (vn-ki) [inline anime_downloader/extractors/vidstream.py:15]: Are all these supported by the same logic? You could just subclass and add different classes, if thats the case

File: anime_downloader/config.py (hunk 1 above)
Comment (Blatzar) [reply anime_downloader/config.py:87]: So you could choose which vidstream server you prefer, they're different in terms of download speeds (and probably sizes + quality)

File: anime_downloader/extractors/vidstream.py (hunk 2 above)
Comment (Blatzar) [reply anime_downloader/extractors/vidstream.py:15]: It just links urls to extractors, I don't see any need for classes.

File: anime_downloader/config.py (hunk 1 above)
Comment (vn-ki) [reply anime_downloader/config.py:87]: You should add this in the docs somewhere and it needs doc comments in the vidstream class.
